### PR TITLE
landing page setup

### PR DESCRIPTION
**Pull request by simonious1989**

### CODE REVIEW

**Comment by simonious1989** [comment]
@flumonion I have created one landing page template which may require additional styling.
The example is two columns, text left and form on the right.

**Comment by flumonion** [comment]
Will make some suggestions/recommendations and commit them to this branch.

**Comment by flumonion** [comment]
Made some changes.
- shortened url
- made page type descriptive
- changed styling in line with other pages

We should:-
- Create partials for shared code (e.g. artticles and video) this will make it easier to add to other templates in future.
- add name tag to netlify forms.
- ensure we are not duplicate already created form partials.
- remove the word "landing" from the form partial, its just a form and can be included anywhere on site, not specifically for landing pages.

**Comment by flumonion** [comment]
@simonious1989 forgot to also include a way to add a "do not index" for crawlers on these pages only...

**Comment by simonious1989** [comment]
@flumonion 

> Create partials for shared code (e.g. artticles and video) this will make it easier to add to other templates in future.
I have created two new partials:
 - video
 - latest-articles 

> add name tag to netlify forms.
I have updated the form to include a form name. 
Should I do this on the other form partials for consistency?

> ensure we are not duplicate already created form partials.
I did create a new form partial and remove the "text" input.

> remove the word "landing" from the form partial, its just a form and can be included anywhere on site, not specifically for landing pages.
I have done this and renamed it to "quick-contact"

> forgot to also include a way to add a "do not index" for crawlers on these pages only...
I looked into that last night and came across the [following approach](https://github.com/dereckcurry/hugo-robots-meta-tags).
I am not sure if this means we need to go and update all pages now with the suggested front matter.

Also if these pages are not being indexed, we can remove the SEO front matter.

**Comment by flumonion** [comment]
> Should I do this on the other form partials for consistency?

Yes

**Comment by simonious1989** [comment]
@flumonion 

I have added a check in the in the head.html for the `robotsNoFollow` param.
have done a check locally and all other pages without this param are not affected, meaning they still retain:

`<META NAME="ROBOTS" CONTENT="INDEX, FOLLOW">`

All forms now have name tags.
Please could you review and merge.

File: themes/s2s/layouts/partials/latest-articles.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<section class="section section--gray section--size4">
```
Comment (flumonion) [inline themes/s2s/layouts/partials/latest-articles.html:1]: In the partial I don't think you should have the wrapper html.
It should only include the cards div onwards.
This means you can list articles in different ways.
i.e. you could have them run across full screen, or smaller etc.

If we wanted to get fancy, we could include [variables](https://gohugo.io/templates/partials/#variable-scoping) in the partials that can be set outside the partial by the page adding the partial. In this example the partial would be in charge of listing articles, the page adding the partial could decide what articles to add.



For example, on one page we could listt

File: themes/s2s/layouts/partials/quick-contact.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<div class="container">
+    <div class="form form--primary form--primary-alt">
+        <form method="post" name="quick-contact" data-netlify="true">
```
Comment (flumonion) [inline themes/s2s/layouts/partials/quick-contact.html:3]: partial should not have "container" class.

**Review by flumonion — CHANGES_REQUESTED** [review]

File: themes/s2s/layouts/partials/quick-contact.html (hunk 2 above)
Comment (simonious1989) [reply themes/s2s/layouts/partials/quick-contact.html]: @flumonion I have removed the "container" class from the partial.

File: themes/s2s/layouts/partials/latest-articles.html (hunk 1 above)
Comment (simonious1989) [reply themes/s2s/layouts/partials/latest-articles.html]: @flumonion I have updated this partial and renamed to `article-card.html`
I can refactor everywhere else where this article section is being used on the site now also.

**Comment by simonious1989** [comment]
@flumonion 

I have made the following changes:

- created a `call.html` partial 
- updated `quick-contact` partial to remove container class
- used .Scratch to get and set the form name in the lp-form template. (I can change this to rather use a variable so it can be set in the page from matter).
- updated the `article.html` partial 
- updated the contact links on the footer
- updated the lp full form copy

**Pull request merged**